README.md

```diff
@@ -36,6 +162,10 @@ npm run generate
 npm run deploy
 ```
 
+## Testing
+
+The Annotation tool uses [Cypress](https://www.cypress.io/) for integration, component, and unit testing. See the [cypress folder](https://github.com/neurobagel/annotation_tool/tree/master/cypress) for tests, custom commands, and test data.
+
 See the [Nuxt documentation](https://nuxtjs.org/deployments/github-pages/) for more details.
 
 ## Special Directories
```

Comment:
I'd say we can get rid of the entire `Special Directories` section as it doesn't really offer anything here. If anyone wanted to get familiar with the app structure, they'll get a better understanding going through the Nuxt docs.